apps/vc-api/src/vc-api/credentials/credentials.service.ts

```diff
@@ -74,21 +74,39 @@ export class CredentialsService implements CredentialVerifier {
       issueDto.options,
       verificationMethod.id
     );
-    return JSON.parse(
-      await issueCredential(
-        JSON.stringify(issueDto.credential),
-        JSON.stringify(proofOptions),
-        JSON.stringify(key)
-      )
-    );
+
+    const result = await issueCredential(
+      JSON.stringify(issueDto.credential),
+      JSON.stringify(proofOptions),
+      JSON.stringify(key)
+    ).catch((err) => {
+      if (typeof err === 'string') {
```

Comment:
IMO, a comment would be helpful here because it's not immediately obvious why `err === 'string'` if, and only if, the error is from `@spruceid/didkit-wasm-node.issueCredential`. I guess it's because if `JSON.stringify` fails (though this is probably really unlikely) for example then the error would be an `object`?
```suggestion
     // assumption is that non-didkit errors will be of object type
      if (typeof err === 'string') {
```

Reply:
@spruceid/didkit-wasm-node throws strings, not Errors if something else is incorrect, not only parsing JSON. This code makes it possible to see a message in the response body.

Reply:
And yes, probably if something seriously goes wrong within @spruceid/didkit-wasm-node.issueCredential it will throw an instance of the Error class.